conftest.py

```diff
@@ -227,19 +227,28 @@ def comment_1(task_1, employee_1):
     return comment
 
 
+@pytest.fixture
+def comment_2(task_1, employee_1):
+    comment = Comment.objects.create(appUser=employee_1,
+                                     task=task_1,
+                                     title="A problem2",
+                                     description="I dont know how2")
+    return comment
```

Comment:
because you don't have any other operation on the comment you can return it like:
```suggestion
    return Comment.objects.create(appUser=employee_1,
                                     task=task_1,
                                     title="A problem2",
                                     description="I dont know how2")
```